Client/core/Graphics/CPrimitive3DBatcher.cpp

```diff
@@ -0,0 +1,169 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        CPrimitive3DBatcher.cpp
+ *  PURPOSE:
+ *
+ *
+ *****************************************************************************/
+#include <StdInc.h>
+#include "CPrimitive3DBatcher.h"
+////////////////////////////////////////////////////////////////
+//
+// CPrimitive3DBatcher::CPrimitive3DBatcher
+//
+//
+//
+////////////////////////////////////////////////////////////////
+CPrimitive3DBatcher::CPrimitive3DBatcher(bool bPreGUI)
+{
+    m_bPreGUI = bPreGUI;
+}
+////////////////////////////////////////////////////////////////
+//
+// CPrimitive3DBatcher::~CPrimitive3DBatcher
+//
+//
+//
+////////////////////////////////////////////////////////////////
+CPrimitive3DBatcher::~CPrimitive3DBatcher(void)
```

Comment:
Remove the destructor from header and source file if you don't do any work here.